src/codegen_call_io.cpp

```
@@ -111,510 +127,518 @@ static const CodeGen::NativeDispatchEntry io_table[] = {
-                codegenError("listen() max_requests must be a positive integer");
+                cg.codegenError("listen() max_requests must be a positive integer");
         }
     }
 
     llvm::Value *portCallback = nullptr;
     if (e.args.size() == 5) {
-        portCallback = emitExpr(*e.args[4]);
-        if (portCallback->getType() != ptrTy_)
-            codegenError("listen() port_callback must be fn(int) -> Unit");
+        portCallback = cg.emitExpr(*e.args[4]);
+        if (portCallback->getType() != cg.ptrTy_)
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)